dist/nanorequest.js

```diff
@@ -3,6 +3,7 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
     return (mod && mod.__esModule) ? mod : { "default": mod };
 };
 Object.defineProperty(exports, "__esModule", { value: true });
+exports.nanorequest = void 0;
```

Comment:
@jgravois did this get added by rebuilding from source?

Reply:
exactly. `npm test` calls `npm run build` internally.